.travis.yml

```diff
@@ -36,7 +36,10 @@ matrix:
         - sudo ssh default -t 'cd /vagrant && sudo make localintegration RUNC_USE_SYSTEMD=yes'
         # same setup but with fs2 driver instead of systemd
         - sudo ssh default -t 'cd /vagrant && sudo make localintegration'
-        # rootless
+        # cgroupv2+systemd (rootless)
+        # NOTE: setenforce 0 is required due to a `machinectl shell` issue: https://bugzilla.redhat.com/show_bug.cgi?id=1788616
```

Comment:
@rhatdan Any chance to take a look at https://bugzilla.redhat.com/show_bug.cgi?id=1788616 ?

Reply:
This should not be recorded in the runc .travis file.  It has nothing to do with this.  There is no expectation that machinectl shell will work with runc.  It only works with systemd-nspawn at this point with a full systemd running inside.

Reply:
machinectl is used for creating user session for testing in rootless.sh

Reply:
Or, is it better to use `ssh rootless@localhost` for creating the session?

Reply:
I would think that is more natural, if you want to match what users really do.

Reply:
updated